hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -464,7 +699,7 @@ async def _wait(
                 if i < 64:
                     i = i + 1
 
-    # FIXME Error if this is called while within a job of the same Batch
+    # FIXME Error if this is called while in a job within the same Batch
```

Comment:
These FIXMEs should never have been in here in the first place. Let's either create an issue or address them in this PR.

Reply:
https://github.com/hail-is/hail/issues/14338